server/app/game/service.py

```diff
@@ -0,0 +1,92 @@
+from datetime import datetime
+from flask import request, jsonify, Blueprint
+
+from app import db
+from . import gamePage
+from app.models.user import User
+from app.models.game import Game
+from app.models.throw import Throw
+from config import generate_http_response, config
+
+
+@gamePage.route('/get_games', methods=["GET"])
```

Comment:
change route from /game/get_games -> /game, couse you use different http methods and we dont need extra urls